databases/week1/homework-queries.sql

```diff
@@ -0,0 +1,80 @@
+-- Find out how many tasks are in the task table
+SELECT
+  COUNT(id)
```

Comment:
You should create a new line per statement, e.g.
`SELECT COUNT(*) as NumberOfTasks`
`FROM task;`